conf/cloud-local.conf

```diff
@@ -27,7 +27,7 @@ pkg_src_maven="https://repo1.maven.org/maven2"
 pkg_accumulo_ver="1.8.1"
 pkg_hbase_ver="1.2.6"
 # Note pkg_spark_hadoop_ver below if modifying
-pkg_hadoop_ver="2.7.4"
+pkg_hadoop_ver="2.7.5"
```

Comment:
:+1:  actually we need to use the "archive" site in general i'll add a ticket for this. That way this would keep working instead of having to upgrade

Reply:
https://github.com/ccri/cloud-local/issues/63

Reply:
@ddseapy yeah, here is the convo